book/src/clauses/well_known_traits.md

```diff
@@ -40,7 +40,7 @@ Some common examples of auto traits are `Send` and `Sync`.
 | immutable refs  | ✅   |  ✅  |   ✅   |  ⚬    |   ⚬   |  ⚬  |   ⚬    |   ⚬      |      ❌      |
 | mutable refs    | ⚬    |  ⚬   |   ✅   |  ⚬    |   ⚬   |  ⚬  |   ⚬    |   ⚬      |      ❌      |
 | slices          | ⚬     | ⚬    | ⚬     |   ✅    | ⚬   | ⚬   | ⚬      |  ⚬       |    ❌       |
-| arrays❌         | ❌     | ❌    | ❌     | ❌      | ⚬   | ⚬   | ⚬      |  ⚬       |    ❌        |
```

Comment:
`Unsize` pr is still pending, ✅ is not correct here

Reply:
Ah yes, that's a mistake. Then the other ✅'s in the column are also incorrect, no?

Reply:
I think it would be easier to just extend this pr so `Sized`, `Copy` and `Clone` cover arrays, it's fairly trivial

Reply:
(should've probably mentioned it in my comment :upside_down_face: )

Reply:
I might be misunderstanding, but the PR already handles the `Sized`, `Copy` and `Clone` clauses for arrays.

Reply:
Uhh, you are correct, sorry, I completely missed it. What other ✅ were you talking about then?

Reply:
No worries. I was talking about the existing ✅ in the "Unsize" column for the other types, for instance for tuples and structs.

Reply:
They are implemented, but not yet merged, so I took the liberty to add them prematurely when writing the book page :)